users/migrations/0003_alter_profile_bio_alter_profile_city_and_more.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2 on 2023-05-03 15:08
```

Comment:
Why is that the name of the file?

Reply:
you are right, you should not, but it `alter_profile_bio_alter_profile_city_and_more` something that was generated automatically? as far as I know migration's auto-generated names are usually just `0003_migration` or something along those lines.

Reply:
I don't know why but this is the file that the migration created for me automatically... in general, I had a lot of headaches with this migration in the last two weeks... I also wrote about it on Slack but there was no response and I would be happy to help. (also relevant regarding the previous comment about the signals -  I try to delete it but only when it's in the comment it works. )